src/FluentValidation/ValidationException.cs

```diff
@@ -49,11 +49,23 @@ public class ValidationException : Exception {
 		public ValidationException(string message, IEnumerable<ValidationFailure> errors) : base(message) {
 			Errors = errors;
 		}
-		/// <summary>
-		/// Creates a new ValidationException
-		/// </summary>
-		/// <param name="errors"></param>
-		public ValidationException(IEnumerable<ValidationFailure> errors) : base(BuildErrorMessage(errors)) {
+
+    /// <summary>
+    /// Creates a new ValidationException
+    /// </summary>
+    /// <param name="message"></param>
+    /// <param name="errors"></param>
+    /// <param name="addErrorMessage"></param>
+    public ValidationException(string message, IEnumerable<ValidationFailure> errors, bool addErrorMessage)

```

Comment:
Could the new parameter be renamed? I think something like `appendDefaultMessage` would be slightly clearer